package.json

```diff
@@ -21,7 +21,7 @@
     "@eeacms/volto-object-widget"
   ],
   "dependencies": {
-    "@eeacms/volto-embed": "*",
+    "@eeacms/volto-embed": "github:eea/volto-embed#develop",
```

Comment:
@razvanMiu Please leave dependency with `*` and pin it in frontend / kitkat, otherwise we'll have resolution issues when deploying :wink: